<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalSAM2</class>
 <widget class="qMRMLWidget" name="MedicalSAM2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>697</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>78</height>
      </size>
     </property>
     <property name="text">
      <string>Backend Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="2">
       <widget class="QPlainTextEdit" name="txtPort">
        <property name="plainText">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="txtIP">
        <property name="plainText">
         <string>IP Address</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_4">
     <property name="text">
      <string>Preprocessing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2" colspan="2">
       <widget class="ctkSliderWidget" name="sldWinWidth">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-2000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>2000.000000000000000</double>
        </property>
        <property name="value">
         <double>400.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="lblWidth">
        <property name="text">
         <string>Window Width:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QPushButton" name="pbApplyPrep">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Preprocessing Options:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="cmbPrepOptions"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="ctkSliderWidget" name="sldWinLevel">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-2000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>2000.000000000000000</double>
        </property>
        <property name="value">
         <double>40.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="lblLevel">
        <property name="text">
         <string>Window Level:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="text">
      <string>ROI</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>Set As Start Slice</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnEnd">
        <property name="text">
         <string>Set As End Slice</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="btnROI">
        <property name="text">
         <string>Add Bounding Box</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_3">
     <property name="text">
      <string>Segmentation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Model Size:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QPushButton" name="btnSegment">
        <property name="text">
         <string>Full Segmention</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="btnMiddleSlice">
        <property name="text">
         <string>Segment Middle Slice</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="btnRefine">
        <property name="text">
         <string>Refine Middle Segmention</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="cmbCheckpoint"/>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="ctkPathLineEdit" name="pathModel">
        <property name="filters">
         <set>ctkPathLineEdit::Executable|ctkPathLineEdit::Files|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
        </property>
        <property name="nameFilters">
         <stringlist>
          <string>*.pt</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
